ui/widgets/calendar.js

```diff
@@ -150,7 +151,14 @@ return $.widget( "ui.calendar", {
 		}
 
 		if ( this._needsRefresh() ) {
-			this._refresh();
+			if ( this.options.numberOfMonths > 1 && this.date.year() === this.viewDate.year() ) {
+				this.viewDate.adjust( "M", this.options.numberOfMonths *
+					( this.date.month() > this.viewDate.month() ? 1 : -1 )
+				);
+				this.refresh();
+			} else {
+				this._refresh();
```

Comment:
>  I can't tell the difference between _refresh() and refresh()

One sets this.viewDate, the other does not. Could be removed by adding ````this.viewDate.setTime( this.date.date().getTime() );```` multiple times. The _refresh had more code in it before but was stripped down over the time, perhaps its time to remove it.

This multi line comment was added loooong time ago. To me it sounds like a "we need to check if the basic idea works at all" comment as refresh is used a lot (basically on every interaction). Anyway, some (not all) of the concerns have been fixed (like changing text instead of regenerate the buttons) lately. I would say we keep this until we fixed some more known issues and tested this for a11y.